app/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ android {
 
     defaultConfig {
         applicationId = "io.homeassistant.companion.android"
-        minSdk = 21
+        minSdk = 23
```

Comment:
This will be a very big breaking change for us as it means the app will no longer be available to these devices, is it absolutely necessary for this feature?

Reply:
The [CarHardware](https://developer.android.com/training/cars/apps#car-hardware-requirements) APIs are provided by the library `androidx.car.app:app-projected` which declares a minSdk of 23 but I guess we could aslo use [overrideLibrary](https://developer.android.com/studio/build/manage-manifests?hl=fr#override_uses-sdk_for_imported_libraries) in the manifest.

Reply:
If we could do that we should as we can hide the sensors from unsupported devices using the `hasSensor` method